.github/workflows/Build.yml

```diff
@@ -25,11 +25,15 @@ jobs:
           - os: windows-2019
```

Comment:
It may be worth increasing the `max-parallel` here to `4` so all jobs run at the same time.  Otherwise, LGTM